Targets/DomainInterface/Sources/Model/LoginModelProtocol.swift

```diff
@@ -13,4 +13,5 @@ public protocol LoginModelProtocol {
     func checkKakaoLogin() async -> Bool
     func isKakaoTalkLoginUrl(_ url: URL) -> Bool
     func authController(url: URL) -> Bool
+    func requestAccessToken() async -> (String, Int)?
```

Comment:
```suggestion
    func requestAccessToken() async -> Bool
```
[용우님 LoginModel 메소드 코멘트](https://github.com/depromeet/ticlemoa-iOS/pull/154/files#r1065182733)에 대한 프로토콜 변경점입니다